frontend/src/App.vue

```diff
@@ -1,19 +1,27 @@
 <script setup lang="ts">
+import { inject } from 'vue'
 import { RouterLink, RouterView } from 'vue-router'
+
+const environmentDisplayName = inject('fam_environment_display_name') as string
+var environmentLabel = ''
```

Comment:
Does compiler complaint with using 'let' not 'var'?
Is 'inject' also get value from 'localstorage'?
Also is it possible for Vue to use 'environmentLable' directly for 'inject' (I mean so we don't need to declare two variables)?